bin/dev/updatetpl.py

```diff
@@ -58,7 +58,7 @@ def help(pname):
 
 for tplpath in tplpaths:
 	print "Converting " + path + tplpath
-	subprocess.call(['python', path + 'util/friendica-to-smarty-tpl.py', '-p', path + tplpath], stdout = fnull)
+	subprocess.call(['python', path + 'bin/dev/friendica-to-smarty-tpl.py', '-p', path + tplpath], stdout = fnull)
```

Comment:
I think we can remove this script as the very old internal templating engine is not supported anymore for some time now as far as I can remember.

Reply:
So both `bin/dev/updatetpl.py` and `bin/dev/friendica-to-smarty-tpl.py`?

Reply:
Yes, I would say so.

Reply:
Thanks!